clippy_lints/src/dereference.rs

```diff
@@ -1014,9 +1014,15 @@ fn report<'tcx>(
                         },
                         _ => (0, false),
                     };
+                    let is_in_tuple = match cx.tcx.parent_hir_node(data.first_expr.hir_id) {
+                        Node::Expr(e) => matches!(e.kind, ExprKind::Tup(_)),
```

Comment:
nit: use `clippy_utils::get_parent_expr(cx, data.first_expr)`

Which does exactly the same thing.

Reply:
(waiting for this)